src/elements/content-picker/stories/tests/ContentPicker-visual.stories.js

```diff
@@ -17,8 +25,708 @@ export const selectedEmptyState = {
         const selectedButton = canvas.getByRole('button', { name: '0 Selected' });
         await userEvent.click(selectedButton);
         await waitFor(() => {
-            expect(canvas.getByText('You haven’t selected any items yet.')).toBeInTheDocument();
+            expect(canvas.getByText("You haven't selected any items yet.")).toBeInTheDocument();
+        });
+    },
+    parameters: {
+        msw: {
+            handlers: [
+                http.get(`${DEFAULT_HOSTNAME_API}/2.0/folders/69083462919`, () => {
+                    return HttpResponse.json(mockRootFolder);
+                }),
+            ],
+        },
+    },
+};
+
+export const emptyFolder = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        await waitFor(
+            () => {
+                // Verify empty folder state using partial match
+                expect(canvas.getByText(/no items/i)).toBeInTheDocument();
+            },
+            {
+                timeout: SLEEP_TIMEOUT * 3, // Increase timeout to allow for retries
+            },
```

Comment:
do you need this timeout anymore? you should get a response back immediately with an empty folder when you use the mockEmptyRootFolder